Lombiq.HelpfulExtensions/Views/Widget-ContainerWidget.cshtml

```diff
@@ -0,0 +1,37 @@
+@using OrchardCore.ContentManagement
+@using OrchardCore.DisplayManagement
+@using OrchardCore.Mvc.Utilities
+@using OrchardCore.Title.Models
+
+@{
+    var title = (Model.ContentItem as ContentItem)?.As<TitlePart>()?.Title;
```

Comment:
You should use DisplayText instead of TitlePart. TitlePart might contain Liquid code. I’m not sure about other reasons, but I’m pretty sure DisplayText is the single source of truth when it comes to the title.